inloop/settings.py

```diff
@@ -206,3 +206,8 @@
 
 CONSTANCE_CONFIG.update(gitload_constance.config)
 CONSTANCE_CONFIG_FIELDSETS.update(gitload_constance.fieldsets)
+
+# Define the output path of the JPlag tests and their similarity factor
+JPLAG_JAR_PATH = str(BASE_DIR / "lib/jplag-2.11.9-SNAPSHOT.jar")
+JPLAG_OUTPUT_PATH = str(BASE_DIR / "media/jplag/output")
```

Comment:
This could be a temporary directory? Because at the end, we are only interested in the ZIP file which now gets saved in the model.